DataFormats/PatCandidates/interface/TauPFEssential.h

```diff
@@ -54,6 +54,7 @@ struct TauPFEssential {
   float hcalEnergy_;
   float leadingTrackNormChi2_;
   float etaAtEcalEntrance_;
+  float phiAtEcalEntrance_;
```

Comment:
You can leave this stuff in but I would not add it to the next PR. We take this one on a separate note.